tests/unit/test_manifest.py

```diff
@@ -55,16 +56,31 @@ def test_manifest_has_deterministic_hash(self) -> None:
     def test_manifest_hash_is_consistent_regardless_of_format(self) -> None:
 
         # Act
+        @dataclass
+        class DateType:
+            """Temp date class for testing"""
+
+            date: datetime
```

Comment:
The tool only works on json which makes sense. This ensures there is an actual class and removes the tech debt around `from_raw`